### PR TITLE
Modified abs, acos, asin math functions  

### PR DESCRIPTION
**Pull request by l2eza**
modified:   content/cpp/concepts/math-functions/terms/abs/abs.md	
        modified:   content/cpp/concepts/math-functions/terms/acos/acos.md
	modified:   content/cpp/concepts/math-functions/terms/asin/asin.md

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@l2eza This looks good, passing it on to a second review.

File: content/cpp/concepts/math-functions/terms/asin/asin.md [hunk 1]
```diff
@@ -37,3 +37,21 @@ int main() {
   // Output: 1.5708 radians
 }
 ```
+
+## Codebyte Example
+
+In this example, the `std::asin()` function is applied to the variable `x`, which holds the value `0.5`. The `std::asin()` function returns the inverse sine (also known as arcsine) of the provided argument. The inverse sine of `0.5` is approximately `0.5236` radians
```
Comment (caupolicandiaz) [inline content/cpp/concepts/math-functions/terms/asin/asin.md:43]:
```suggestion
In this example, the `std::asin()` function is applied to the variable `x`, which holds the value `0.5`. The `std::asin()` function returns the inverse sine (also known as arcsine) of the provided argument.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@l2eza I've added a few comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by SSwiniarski** [comment]
@l2eza are you going to address @caupolicandiaz's comments, it's been three weeks. If we don't see any movement in the next couple of days we'll close this PR due to inactivity.

**Comment by SSwiniarski** [comment]
Closing this PR due to inactivity.

**Pull request closed without merging**